containers/kubernetes-helm-k3d/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Note: You can use any Debian/Ubuntu based image you want. 
+FROM mcr.microsoft.com/vscode/devcontainers/base:0-bullseye
+
+# Enable new "BUILDKIT" mode for Docker CLI
+ENV DOCKER_BUILDKIT=1
+
+# Options
+ARG INSTALL_ZSH="true"
+ARG UPGRADE_PACKAGES="false"
+ARG USE_MOBY="true"
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+# Install needed packages and setup non-root user. Use a separate RUN statement to add your own dependencies.
+COPY library-scripts/*.sh /tmp/library-scripts/
+RUN apt-get update && /bin/bash /tmp/library-scripts/common-debian.sh "${INSTALL_ZSH}" "${USERNAME}" "${USER_UID}" "${USER_GID}" "${UPGRADE_PACKAGES}" "true" "true" \
+    && /bin/bash /tmp/library-scripts/docker-in-docker-debian.sh "true" "${USERNAME}" "${USE_MOBY}" \ 
+    && /bin/bash /tmp/library-scripts/kubectl-helm-debian.sh "latest" "latest" "latest" \
+    && mkdir -p /home/${USERNAME}/.k3d \
```

Comment:
I'm not 100% that `/home` should be hardcoded here. It might be that the user's home dir not to be at `/home`. I think it can be configurable on devcontainer.json

Reply:
We followed the same pattern used by kubernetes-helm-minikube.
But using $HOME is better since, it should be set per-user with `useradd` or `usermod` command.